packages/ckeditor5-mention/src/mentionui.ts

```diff
@@ -239,7 +241,7 @@ export default class MentionUI extends Plugin {
 			const { item, marker } = data;
 
 			// Set to 10 by default for backwards compatibility. See: #10479
-			const dropdownLimit = this.editor.config.get( 'mention.dropdownLimit' ) || 10;
+			const dropdownLimit = data.dropdownLimit || this.editor.config.get( 'mention.dropdownLimit' ) || 10;
```

Comment:
I think we don't need to pass the `dropdownLimit` all over the place. It would be much simpler (and would affect less code) if you would resolve the limit depending on the `marker` here. You could normalize the config to store limits for specific markers and just access this map here.

Reply:
c2b3c7b

I have extend the `_mentionsConfigurations` to pass there the `dropdownLimit` . I think thatthis is the right way 🙂